target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -391,7 +391,8 @@ ramips_setup_interfaces()
 		;;
 	vonets,var11n-300|\
 	wt1520-4M|\
-	wt1520-8M)
+	wt1520-8M|\
+	fon,fon2601)
```

Comment:
Please change this. The entries should be in lexicographical order.

Reply:
changed at def4fb1